modules/graphql_metatag/src/Plugin/GraphQL/Fields/MetaValue.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Drupal\graphql_metatag\Plugin\GraphQL\Fields;
+
+use Drupal\graphql_core\GraphQL\FieldPluginBase;
+use Youshido\GraphQL\Execution\ResolveInfo;
+
+/**
+ * The metatags content field.
+ *
+ * @GraphQLField(
+ *   id = "meta_value",
+ *   name = "value",
+ *   type = "String",
+ *   types = {"MetaTag"}
+ * )
+ */
+class MetaValue extends FieldPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function resolveValues($value, array $args, ResolveInfo $info) {
+    if (is_array($value) && array_key_exists('value', $value)) {
```

Comment:
I don't know... I find this weak type checking weird. Can we use a container object for $value?